### ISSUE TITLE
is_training not set in one of the layer of vladder_small

### ISSUE THREAD

**Issue by l3robot** [ShengjiaZhao/Variational-Ladder-Autoencoder#1]
Hi, very interesting paper btw! 
I'm replicating the code in PyTorch, but noticed that you've not used the `is_training` parameter for one of the layer below. I was wandering if that was intended, since some papers are keeping the training behaviour of batch normalization in evaluation mode.
https://github.com/ermongroup/Variational-Ladder-Autoencoder/blob/00d517dd3c7baa341c83298a58c898b3a8267db7/vladder_small.py#L66
Thanks

**Comment by DanRuta** [comment]
@l3robot Have you had success replicating the code in Pytorch? If so, is there any chance it could be made public?
